cmd/zoekt-sourcegraph-indexserver/debug.go

```diff
@@ -147,6 +154,98 @@ func debugListIndexed() *ffcli.Command {
 	}
 }
 
+func debugQueue() *ffcli.Command {
+	var connectionTimeout time.Duration
+	var printHeader bool
+
+	var serverAddress string
+
+	fs := flag.NewFlagSet("debug queue", flag.ExitOnError)
+	fs.DurationVar(&connectionTimeout, "timeout", 10*time.Second, "max timeout for establishing a connection to a zoekt-sourcegraph-indexserver instance")
+	fs.BoolVar(&printHeader, "header", false, "whether to print the headers for each column")
+
+	fs.StringVar(&serverAddress, "address", "http://localhost:6072", "the address of the zoekt-sourcegraph-indexserver instance to connect to")
+
+	return &ffcli.Command{
+		Name:       "queue",
+		ShortUsage: "queue [flags]",
+		ShortHelp:  "list the repositories in the indexing queue, sorted by descending priority",
+		FlagSet:    fs,
+		Exec: func(ctx context.Context, args []string) error {
+
+			fullAddress := fmt.Sprintf("%s/debug/queue", serverAddress)
```

Comment:
I think we should use url.Parse to parse the serverAddress. Go's next release seems to come with a handy function for joining https://sourcegraph.com/github.com/golang/go/-/blob/src/net/url/url.go?L1193&subtree=true